crates/stark_curve/src/field.rs

```diff
@@ -6,8 +6,6 @@ use bitvec::{array::BitArray, order::Lsb0};
 use ff::PrimeField;
 
 /// The field primitive used by Starkware's curve.
-///
-/// It's main use is to allow [`pedersen_hash`](crate::hash::pedersen_hash).
```

Comment:
I think comments like this will help learnability, you can start wondering from top or bottom and still understand how the pieces fit. I wouldn't remove it just because one cannot link after separation, could do a non-link "link".